coremltools/converters/mil/frontend/torch/test/test_executorch_quantization.py

```diff
@@ -0,0 +1,56 @@
+#  Copyright (c) 2023, Apple Inc. All rights reserved.
+#
+#  Use of this source code is governed by a BSD-3-clause license that can be
+#  found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+
+import pytest
+
+import torch
+from torch._export import capture_pre_autograd_graph
```

Comment:
we are calling this `capture_pre_autograd_graph` through `torch._export`,
in which is a non public API and tend to be changed,
so it would be good not to make our code depends on those methods.

Reply:
It's PyTorch 2 export [quantization API](https://pytorch.org/docs/stable/quantization.html#prototype-pytorch-2-export-quantization), so we have no choice but to depend on it for testing 😞 even if PyTorch folks do declare it as prototype